specification/resources/resource-manager/Microsoft.Management/preview/2018-01-01-preview/management.json

```diff
@@ -0,0 +1,1266 @@
+      "properties": {
+        "type": {
+          "title": "The type of child resource.",
+          "$ref": "#/definitions/ManagementGroupChildType",
+          "description": "The fully qualified resource type which includes provider namespace (e.g. /providers/Microsoft.Management/managementGroups)"
+        },
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the child resource (management group or subscription).  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the child entity."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the child resource."
+        },
+        "roles": {
+          "description": "The roles associated with the management group.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "children": {
+          "description": "The list of children.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagementGroupChildInfo"
+          }
+        }
+      }
+    },
+    "ParentGroupInfo": {
+      "description": "(Optional) The ID of the parent management group.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the parent management group.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000"
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the parent management group"
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The friendly name of the parent management group."
+        }
+      }
+    },
+    "ManagementGroupChildType": {
+      "type": "string",
+      "enum": [
+        "/providers/Microsoft.Management/managementGroups",
+        "/subscriptions"
+      ],
+      "description": "The type of child resource."
+    },
+    "OperationResults": {
+      "description": "The results of an asynchronous operation.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The fully qualified ID for the management group.  For example, /providers/Microsoft.Management/managementGroups/0000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the resource.  For example, /providers/Microsoft.Management/managementGroups",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the management group. For example, 00000000-0000-0000-0000-000000000000",
+          "readOnly": true
+        },
+        "provisioningState": {
```

Comment:
>provisioningState [](start = 9, length = 17)

Could this be an enum?

Reply:
Right now it's just one value...  But we can change it